test/parallel/test-whatwg-encoding-textdecoder-streaming.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+// From: https://github.com/w3c/web-platform-tests/blob/fa9436d12c5edb21704c3382e299aaac31683b24/encoding/textdecoder-streaming.html
+
+const common = require('../common');
+
+if (!common.hasIntl)
+  common.skip('missing Intl');
```

Comment:
With the exception of UTF-16BE, this test should work even when ICU is not enabled.